### PR TITLE
Use CP 4.1.0 docker images for 4.1 branches

### PR DESCRIPTION
**Pull request by miguno**
**Important:** This PR must only be merged once we published a 4.1.0 tagged image for https://hub.docker.com/r/confluentinc/kafka-streams-examples/.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 version: '2'
 services:
   zookeeper:
-    image: confluentinc/cp-zookeeper:latest
+    image: confluentinc/cp-zookeeper:4.1.0
```
Comment (miguno) [inline docker-compose.yml]: Because of Docker's counter-intuitive behavior for "latest", we should always specify the exact version of the image.

**Comment by apurvam** [comment]
It doesn't look like the image is there though, so this still should not be merged.

**Comment by mjsax** [comment]
@miguno LGTM -- can you merge this?

**Comment by miguno** [comment]
@mjsax : No, I can't merge this yet.  There is still no `4.1.0` tagged Docker image for kafka-streams-examples, and this is a blocking this PR.

**Comment by guozhangwang** [comment]
@miguno this PR can be merged as we have new tags now: https://hub.docker.com/r/confluentinc/kafka-streams-examples/tags/

**Comment by miguno** [comment]
Also pint-merged.

**Pull request merged**